src/Logo/index.js

```diff
@@ -30,8 +30,8 @@ LogoIcon.defaultProps = {
 }
 
 LogoIcon.propTypes = {
-    className: PropTypes.string,
-    fill: PropTypes.string,
+    className: propTypes.string,
+    fill: propTypes.string,
```

Comment:
I don't think people should be allowed to specify any fill here. According to the DHIS2 identity, we have 4 versions of the logo. I think this component should reflect that.
- Icon only VS icon with text
- Blue on white VS white on blue

Reply:
Agreed, I've improved the prop names and locked the external API.

4371430